src/main/java/org/commonjava/util/partyline/impl/infinispan/model/InfinispanJFS.java

```diff
@@ -168,11 +185,25 @@ public void onCacheCreatedEvent( CacheEntryEvent event )
             if ( event.getKey() == this.key )
             {
                 // TODO: Do something here to notify
-                return;
+                this.lock.signal();
             }
             events.add( event );
         }
 
+        @CacheEntryModified
+        public void onCacheEventModified( CacheEntryEvent event )
+        {
+            if ( event.getKey() == this.key )
```

Comment:
I think we want to use .equals() instead of == here, since deserialization will probably lead to different physical instances being compared.

Reply:
I thought we would want to check for equality of the id itself

https://www.tutorialspoint.com/java/util/uuid_equals.htm

Reply:
comment above...we'll switch to String fields. Resolving this conversation to de-duplicate